gradle.properties

```diff
@@ -1,3 +1,3 @@
 group = org.threadly
-version = 2.0.0-SNAPSHOT
+version = 2.0.1-SNAPSHOT
```

Comment:
What's the point of incrementing a snapshot version?  I would just leave it on 2.0.0